src/sentry/options/defaults.py

```diff
@@ -688,3 +688,8 @@
 # the number of large transactions to retrieve from Snuba for transaction re-balancing
 register("dynamic-sampling.prioritise_transactions.num_explicit_small_transactions", 0)
 register("hybrid_cloud.outbox_rate", default=0.0)
+
+register("test_key.1", default="value1")
+register("test_key.2", default="value2")
+register("test_key.3", default="value3")
+register("test_key.4", default="value4")
```

Comment:
Please do not register options for tests in the main options file. These would be very confusing for people trying to make sense of existing options used in prod. 
If you to switch options values in a test case, please either register them before in the test code before loading up sentry, or just use existing ones. Since you are not testing register/unregister you should not need new options.